test/runtests.jl

```diff
@@ -541,6 +541,40 @@ let
     @test_approx_eq_eps getValue(z) 20 1e-6
 end
 
+# special cases when cc becomes linear
+let 
+    m = ChanceModel()
+    @defIndepNormal(m, x[1:2], mean=1, var=1)
+    @defVar(m, z <= 100)
+    @defVar(m, y[1:2])
+
+    @setObjective(m, Max, z)
+    @addConstraint(m, y[1] == y[2])
+    @addConstraint(m, z + sum{x[i]*y[i], i=1:2} <= 20, with_probability=0.95)
+
+
+    status = solve(m, method=:Reformulate)
```

Comment:
Sounds good. I'd like to merge this separately from any work on the two-sided chance constraints.